spread.yaml

```diff
@@ -74,7 +79,7 @@ prepare: |
 restore-each: |
   # Cleanup after each task.
   docker system prune -a -f
-  lxc --project=rockcraft stop --all
+  # lxc --project=rockcraft stop --all
```

Comment:
how come?

Reply:
that's a leftover. I do this locally cause for some reason, my spread tests fail to stop LXC instances. fixing it